templates/openstack/install.html

```diff
@@ -254,175 +276,757 @@ <h3>Installation instructions</h3>
+Enter credential name: <strong>maas</strong>
+Regions
+default
+Select region [any region, credential is not region specific]: <strong>default</strong>
+Using auth-type "oauth1".
+Enter maas-oauth:
+Credential "maas" added locally for cloud "maas".
+          </code></pre>
+          <p>At this point you can bootstrap the Juju controller on the MAAS cloud:</p>
+          <div class="p-code-copyable">
+            <input class="p-code-copyable__input" value="juju bootstrap maas maas-controller"
+              readonly="readonly">
+            <button class="p-code-copyable__action">Copy to clipboard</button>
+          </div>
+          <p>This command takes a while. Once finished, add a model for your OpenStack
+          installation:</p>
+          <div class="p-code-copyable">
+            <input class="p-code-copyable__input" value="juju add-model openstack"
+              readonly="readonly">
+            <button class="p-code-copyable__action">Copy to clipboard</button>
+          </div>
+          <p>At this point, Juju is ready to deploy OpenStack on the remaining four
+          machines.</p>
         </div>
       </li>
 
       <li class="p-stepped-list__item">
-        <h4 class="p-stepped-list__title"><span class="p-stepped-list__bullet">10</span>Add MAAS endpoint and credentials</h4>
+        <h4 class="p-stepped-list__title">Deploy OpenStack</h4>
         <div class="p-stepped-list__content">
-          <p>The MAAS REST API endpoint will be of the form: <code>http://&lt;maas.ip&gt;/MAAS/</code>.</p>
-          <p>The api key is found under the MAAS &lsquo;admin&rsquo; acouunt page as shown below.</p>
+          <p>In this example, we are using the <a href="https://jaas.ai/openstack-base/bundle/65" class="p-link--external">OpenStack Base</a> bundle which provides basic
+          OpenStack services and no HA. If you need more advanced architecture, we
+          encourage you to <a href="/openstack/contact-us" class="js-invoke-modal">contact Canonical</a>.</p>
+          <p>Run the following command to deploy OpenStack Base bundle:</p>
+          <div class="p-code-copyable">
+            <input class="p-code-copyable__input" value="juju deploy cs:bundle/openstack-base"
+              readonly="readonly">
+            <button class="p-code-copyable__action">Copy to clipboard</button>
+          </div>
+          <p>This command takes a while. Once finished you should see the following output
+          on the bottom:</p>
+          <pre><code>Deploy of bundle completed.</code></pre>
+          <p>But it does not mean that your OpenStack deployment is done. In fact, it has
+          just started. You have to monitor the status by running juju status command.
+          The deployment is completed once all applications, units and machines turn to
+          the &ldquo;active&rdquo; state as depicted on the listing below:</p>
+          <pre><code>$ juju status
+Model Controller Cloud/Region Version SLA Timestamp
```

Comment:
White space formatting has been lost here, making the output look very noisy - lines 800 to 885 should be replaced with:

```
Model      Controller       Cloud/Region  Version  SLA          Timestamp
openstack  maas-controller  maas/default  2.7.0    unsupported  12:22:38-05:00

App                    Version  Status  Scale  Charm                  Store       Rev  OS      Notes
ceph-mon               14.2.2   active      3  ceph-mon               jujucharms   44  ubuntu  
ceph-osd               14.2.2   active      3  ceph-osd               jujucharms  294  ubuntu  
ceph-radosgw           14.2.2   active      1  ceph-radosgw           jujucharms  283  ubuntu  
cinder                 15.0.0   active      1  cinder                 jujucharms  297  ubuntu  
cinder-ceph            15.0.0   active      1  cinder-ceph            jujucharms  251  ubuntu  
glance                 19.0.0   active      1  glance                 jujucharms  291  ubuntu  
keystone               16.0.0   active      1  keystone               jujucharms  309  ubuntu  
mysql                  5.7.20   active      1  percona-cluster        jujucharms  281  ubuntu  
neutron-api            15.0.0   active      1  neutron-api            jujucharms  282  ubuntu  
neutron-gateway        15.0.0   active      1  neutron-gateway        jujucharms  276  ubuntu  
neutron-openvswitch    15.0.0   active      3  neutron-openvswitch    jujucharms  269  ubuntu  
nova-cloud-controller  20.0.0   active      1  nova-cloud-controller  jujucharms  339  ubuntu  
nova-compute           20.0.0   active      3  nova-compute           jujucharms  309  ubuntu  
ntp                    3.2      active      4  ntp                    jujucharms   36  ubuntu  
openstack-dashboard    16.0.0   active      1  openstack-dashboard    jujucharms  297  ubuntu  
placement              2.0.0    active      1  placement              jujucharms    1  ubuntu  
rabbitmq-server        3.6.10   active      1  rabbitmq-server        jujucharms   97  ubuntu  

Unit                      Workload  Agent  Machine  Public address  Ports              Message
ceph-mon/0                active    idle   1/lxd/0  172.16.7.164                       Unit is ready and clustered
ceph-mon/1                active    idle   2/lxd/1  172.16.7.163                       Unit is ready and clustered
ceph-mon/2*               active    idle   3/lxd/0  172.16.7.165                       Unit is ready and clustered
ceph-osd/0*               active    idle   1        172.16.7.160                       Unit is ready (1 OSD)
ceph-osd/1                active    idle   2        172.16.7.161                       Unit is ready (1 OSD)
ceph-osd/2                active    idle   3        172.16.7.162                       Unit is ready (1 OSD)
ceph-radosgw/0*           active    idle   0/lxd/0  172.16.7.166    80/tcp             Unit is ready
cinder/0*                 active    idle   1/lxd/2  172.16.7.168    8776/tcp           Unit is ready
  cinder-ceph/0*          active    idle            172.16.7.168                       Unit is ready
glance/0*                 active    idle   2/lxd/1  172.16.7.167    9292/tcp           Unit is ready
keystone/0*               active    idle   3/lxd/2  172.16.7.169    5000/tcp           Unit is ready
mysql/0*                  active    idle   0/lxd/2  172.16.7.170    3306/tcp           Unit is ready
neutron-api/0*            active    idle   1/lxd/2  172.16.7.171    9696/tcp           Unit is ready
neutron-gateway/0*        active    idle   0        172.16.7.159                       Unit is ready
  ntp/0*                  active    idle            172.16.7.159    123/udp            chrony: Ready
nova-cloud-controller/0*  active    idle   2/lxd/2  172.16.7.172    8774/tcp,8775/tcp  Unit is ready
nova-compute/0*           active    idle   1        172.16.7.160                       Unit is ready
  neutron-openvswitch/2   active    idle            172.16.7.160                       Unit is ready
  ntp/3                   active    idle            172.16.7.160    123/udp            chrony: Ready
nova-compute/1            active    idle   2        172.16.7.161                       Unit is ready
  neutron-openvswitch/1   active    idle            172.16.7.161                       Unit is ready
  ntp/2                   active    idle            172.16.7.161    123/udp            chrony: Ready
nova-compute/2            active    idle   3        172.16.7.162                       Unit is ready
  neutron-openvswitch/0*  active    idle            172.16.7.162                       Unit is ready
  ntp/1                   active    idle            172.16.7.162    123/udp            chrony: Ready
openstack-dashboard/0*    active    idle   3/lxd/2  172.16.7.174    80/tcp,443/tcp     Unit is ready
placement/0*              active    idle   2/lxd/3  172.16.7.175    8778/tcp           Unit is ready
rabbitmq-server/0*        active    idle   0/lxd/2  172.16.7.176    5672/tcp           Unit is ready

Machine  State    DNS           Inst id              Series  AZ       Message
0        started  172.16.7.159  openstack1           bionic  default  Deployed
0/lxd/0  started  172.16.7.166  juju-592de2-0-lxd-3  bionic  default  Container started
0/lxd/1  started  172.16.7.170  juju-592de2-0-lxd-4  bionic  default  Container started
0/lxd/2  started  172.16.7.176  juju-592de2-0-lxd-5  bionic  default  Container started
1        started  172.16.7.160  openstack2           bionic  default  Deployed
1/lxd/0  started  172.16.7.164  juju-592de2-1-lxd-3  bionic  default  Container started
1/lxd/1  started  172.16.7.168  juju-592de2-1-lxd-4  bionic  default  Container started
1/lxd/2  started  172.16.7.171  juju-592de2-1-lxd-5  bionic  default  Container started
2        started  172.16.7.161  openstack3           bionic  default  Deployed
2/lxd/0  started  172.16.7.163  juju-592de2-2-lxd-4  bionic  default  Container started
2/lxd/1  started  172.16.7.167  juju-592de2-2-lxd-5  bionic  default  Container started
2/lxd/2  started  172.16.7.172  juju-592de2-2-lxd-6  bionic  default  Container started
2/lxd/3  started  172.16.7.175  juju-592de2-2-lxd-7  bionic  default  Container started
3        started  172.16.7.162  openstack4           bionic  default  Deployed
3/lxd/0  started  172.16.7.165  juju-592de2-3-lxd-3  bionic  default  Container started
3/lxd/1  started  172.16.7.169  juju-592de2-3-lxd-4  bionic  default  Container started
3/lxd/2  started  172.16.7.174  juju-592de2-3-lxd-5  bionic  default  Container started
```